scripts/asciidoc/links.txt

```diff
@@ -200,7 +200,7 @@ Alc-ToServer-Dhcp-Options https://infoproducts.alcatel-lucent.com/html/0_add-h-f
 Alc-ToClient-Dhcp-Options https://infoproducts.alcatel-lucent.com/html/0_add-h-f/93-0088-HTML/7750_SR_OS_Radius_Attributes_Reference_Guide/SROS_RADIUS_Attrib.html
 draft-sterman-aaa-sip-01 https://tools.ietf.org/html/draft-sterman-aaa-sip-01
 draft-wahl-ldap-session https://tools.ietf.org/html/draft-wahl-ldap-session-03
-PAM http://www.kernel.org/pub/linux/libs/pam/index.html
+PAM https://cdn.kernel.org/pub//linux/libs/pam/
```

Comment:
I don't think this is the same thing.  the `www` link is the authoritative source.  The `cdn` link is the _current_ content distribution network.  And it may change.

Reply:
I thought the `www` link doesn't work, but it's just the `index.html` file that doesn't work. I'll fix that.